tests/test_shacl.py

```diff
@@ -8,6 +8,7 @@
         ("Catalog", "example-catalog"),
         ("Dataset", "example-dataset"),
         ("Distribution", "example-distribution"),
+        ("DataService","example-dataservice"),
```

Comment:
```suggestion
        ("DataService", "example-dataservice"),
```

@Markus92 do we need a python linter on test code you think?